packages/grafana-ui/src/components/Alert/Alert.tsx

```diff
@@ -122,20 +123,18 @@ export const getIconFromSeverity = (severity: AlertVariant): IconName => {
 };
 
 const getStyles = (
-  theme: GrafanaTheme2,
+  theme: GrafanaTheme3,
   severity: AlertVariant,
   hasTitle: boolean,
   elevated?: boolean,
   bottomSpacing?: number,
   topSpacing?: number
 ) => {
-  const color = theme.colors[severity];
-
   return {
     wrapper: css({
       flexGrow: 1,
-      marginBottom: theme.spacing(bottomSpacing ?? 2),
-      marginTop: theme.spacing(topSpacing ?? 0),
+      marginBottom: bottomSpacing ? `calc(${bottomSpacing} * ${theme.dimension}` : theme.spacing.x200,
```

Comment:
Am I missing smth or is the `calc` function here missing the closing parenthesis? I'm also wondering if we'd have some spacing function even with the token system to abstract this calculation logic?